backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/impl/IndividualAchievementsServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.example.BackendMIIT.service.impl;
+
+import com.example.BackendMIIT.model.domain.IndividualAchievements;
+import com.example.BackendMIIT.parser.IndividualAchievementsParser;
+import com.example.BackendMIIT.repositories.IndividualAchievementsRepository;
+import com.example.BackendMIIT.service.IndividualAchievementsService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Set;
+
+@Service
+public class IndividualAchievementsServiceImpl implements IndividualAchievementsService {

```

Comment:
unite 3 methods of this class to single

Reply:
What 3 methods are you meaning? I united the parse method and the save method

Reply:
Ok